modis_tools/granule_handler.py

```diff
@@ -176,15 +177,19 @@ def _get_location(url: HttpUrl, modis_session: ModisSession) -> str:
         session = modis_session.session
         split_result = urlsplit(url)
         https_url = split_result._replace(scheme="https").geturl()
-        location_resp = session.get(https_url, allow_redirects=False)
-        if location_resp.status_code == 401:
-            # try using ProxyAuth if BasicAuth returns 401 (unauthorized)
-            location_resp = session.get(
-                https_url,
-                allow_redirects=False,
-                auth=HTTPProxyAuth(modis_session.username, modis_session.password),
-            )
-        location = location_resp.headers.get("Location")
+        if url.host == "ladsweb.modaps.eosdis.nasa.gov":
```

Comment:
```suggestion
        if url.host == URLs.LAADS_RESOURCE.value:
```

Reply:
Makes a lot of sense, thanks for the suggestion!